Repository/ServiceEntityRepository.php

```diff
@@ -8,46 +8,8 @@
 
 if (property_exists(EntityRepository::class, '_entityName')) {
     // ORM 2
-    /**
-     * Optional EntityRepository base class with a simplified constructor (for autowiring).
-     *
-     * To use in your class, inject the "registry" service and call
-     * the parent constructor. For example:
-     *
-     * class YourEntityRepository extends ServiceEntityRepository
-     * {
-     *     public function __construct(ManagerRegistry $registry)
-     *     {
-     *         parent::__construct($registry, YourEntity::class);
-     *     }
-     * }
-     *
-     * @template T of object
-     * @template-extends LazyServiceEntityRepository<T>
-     */
-    class ServiceEntityRepository extends LazyServiceEntityRepository
-    {
-    }
+    class_alias(ServiceEntityRepositoryOrm2::class, 'ServiceEntityRepository');
```

Comment:
Defining only class_alias in the file means that the Composer class loader won't discover the class name when generating an optimized class loader